sdcm/utils/quota.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import absolute_import, annotations
+
+import logging
+import time
+from sdcm.exceptions import QuotaConfigurationFailure
+from sdcm.sct_events import Severity
+from sdcm.sct_events.filters import EventsSeverityChangerFilter
+from sdcm.sct_events.loaders import CassandraStressEvent
+from sdcm.wait import wait_for
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def enable_quota_on_node(node):
+    if not verify_quota_is_enabled_on_node(node):
+        LOGGER.info('Enabling quota on node {}'.format(node.name))
+        change_default_grub_cmd = 'sed -i s\'/GRUB_CMDLINE_LINUX_DEFAULT="quiet splash"/' \
+                                  'GRUB_CMDLINE_LINUX_DEFAULT="quiet splash rootflags=uquota,pquota"/\' ' \
+                                  '/etc/default/grub'
+        mk_grub_cmd = 'grub-mkconfig -o /boot/grub/grub.cfg'
+        enable_uquota_on_mount_cmd = 'sed -i s\'/Options=noatime,discard/Options=noatime,discard,uquota/\' ' \
+                                     '/etc/systemd/system/var-lib-scylla.mount'
+
+        node.remoter.sudo(cmd=change_default_grub_cmd)
+        node.remoter.sudo(cmd=mk_grub_cmd)
+        node.remoter.sudo(cmd=enable_uquota_on_mount_cmd)
+        LOGGER.debug('Rebooting node: "{}"'.format(node.name))
+        node.reboot(hard=False)
+        node.wait_node_fully_start()
+
+
+def verify_quota_is_enabled_on_node(node):
```

Comment:
`verify_` in the name points to verification -> should not return anything (like it doesn't `verify_quota_is_configured_for_scylla_user`)
To keep consistency in naming, use `is_quota_enabled_on_node` for returning quota status